frontend_/src/components/NavTop/NavTop.jsx

```diff
@@ -0,0 +1,13 @@
+import NavTopView from "./NavTopView";
+
+/**
+ *
+ * @param {object} props
+ * @param {string} props.title
+ * @param {string} props.logo
```

Comment:
Also we need some logic to change logo for "go back" Link or button which force Router to navigate back.

We can and should hardcode this logo icon inside NavView

Reply:
I will add it later